packages/error-reporting/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
All of our sub-modules define a `publish-module` npm run script:

```json
  "scripts": {
    "publish-module": "node ../../scripts/publish.js error-reporting"
  }
```

Take a look at [`publish.js`](https://github.com/GoogleCloudPlatform/google-cloud-node/blob/master/scripts/publish.js) to see if it's useful-- I believe it should be. One of the things it does is copy over `AUTHORS`, `CONTRIBUTORS`, and `COPYING` (which is a weird name for this same `LICENSE` file). That means you won't have to keep these files inside `packages/error-reporting`.

It also follows a uniform git tagging system and commit message pattern, used like this:

```sh
$ cd packages/error-reporting
$ npm run publish-module 0.1.0
```